daprdocs/content/en/reference/components-reference/supported-locks/redis-lock.md

```diff
@@ -0,0 +1,115 @@
+---
+type: docs
+title: "Redis"
+linkTitle: "Redis"
+description: Detailed information on the Redis lock component
+---
+
+## Component format
+
+To set up the Redis lock, create a component of type `lock.redis`. See [this guide]({{< ref "howto-use-distributed-lock" >}}) on how to create a lock.
+
+```yaml
+apiVersion: dapr.io/v1alpha1
+kind: Component
+metadata:
+  name: <NAME>
+  namespace: <NAMESPACE>
+spec:
+  type: lock.redis
+  version: v1
+  metadata:
+  - name: redisHost
+    value: <HOST>
+  - name: redisPassword
+    value: <PASSWORD>
+
+
+```
+
+{{% alert title="Warning" color="warning" %}}
+The above example uses secrets as plain strings. It is recommended to use a secret store for the secrets, as described [here]({{< ref component-secrets.md >}}).
+{{% /alert %}}
+
+
+## Spec metadata fields
```

Comment:
All the metadata fields of `state.redis` apply here. Same for the YAML example above for `Component format`.

Reply:
Pulled in all relevant fields